metadata-ingestion/tests/unit/test_transform_dataset.py

```diff
@@ -775,14 +775,19 @@ def test_simple_dataset_tags_transformation(mock_time):
             ]
         )
     )
-    assert len(outputs) == 3
+
+    assert len(outputs) == 5
```

Comment:
if len is now 5, can we add a test for `outputs[4]` too?

Reply:
done